docs/wxm-devices/helium-bundle/helium-connect-and-claim.mdx

```diff
@@ -14,6 +14,8 @@ In this chapter, you will find guidance to set up the WeatherXM Helium station.
 
 1. The station must be fully assembled (parts and batteries)
 1. A working smartphone with connection to the internet
+1. The WeatherXM app installed on your smartphone. [WeatherXM Application](/wxm-devices/D1-bundle/D1-connect-and-claim#download-weatherxm-app-and-login)
```

Comment:
Wrong link, it goes to D1 not the Helium's one! Also, it doesn't make sense as it is now that a "WeatherXM Application" is after the dot. Please make it more human-friendly.